tests/conftest.py

```diff
@@ -0,0 +1,192 @@
+import os
+import sys
+import json
+import pytest
+import asyncio
+from binascii import hexlify
+
+import urllib.parse
+import tornado
+from tornado.escape import url_escape
+
+from traitlets.config import Config
+
+import jupyter_core.paths
+from jupyter_server.serverapp import ServerApp
+from jupyter_server.utils import url_path_join
+
+
+pytest_plugins = ("pytest_tornasync")
+
+
+@pytest.fixture
+def asyncio_patch():
+    ServerApp()._init_asyncio_patch()
+
+
+@pytest.fixture
+def io_loop(asyncio_patch):
```

Comment:
I think it might be a good idea to add a comment here that this fixture (sans the asyncio_patch reference) comes directly from plugin `pytest-tornasync` and that we need to override fixture `io_loop` in order to ensure the appropriate event loop policy is in place and that these (asyncio_patch and _this_ io_loop) fixtures can be removed once the ProactorEventLoop works correctly on Windows (and pytlhon 3.8+).

Reply:
:+1: